packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -307,7 +308,7 @@ class ScrollPosition extends ViewportOffset {
 
   @override
   void correctBy(double correction) {
-    _pixels += correction;
+    correctPixels(_pixels + correction);
```

Comment:
OK, I'll take it out.  An advantage of the override, as things stand now, is that if mySubclass.correctPixels() has a side-effect, then the subclass can short-circuit if there's no actual change.